### PR TITLE
adjusting to new frism plans

### PR DESCRIPTION
**Pull request by haitamlaarabi**
Quick fix to FRISM plans reader

- - -
This change is [Reviewable](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3894)

### CODE REVIEW

**Comment by haitamlaarabi** [comment]
Test now!

File: src/main/python/freight/freight_events_processing.py [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+    plt.tight_layout()
+    png_output = os.path.join(get_local_work_directory(scenario), f"vmt_by_category_{scenario}.png")
+    plt.savefig(png_output)
+
+    return pt
+
+
+def convert_payload_to_trips(payload_df, scenario):
+    output_file_path = os.path.join(work_dir, "beam-freight", batch, scenario,
+                                    f"trips--{scenario.replace("_", "-")}.csv")
```
Comment (dimaopen) [inline src/main/python/freight/freight_events_processing.py]: Older version of python (i.e. 3.10) don't support nesting same quoted strings. I suggest replace nested double-quoted strings to single quoted ones. It helps avoiding issues when running this script in other environments.

**Review by dimaopen — APPROVED** [review]

**Review by haitamlaarabi — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3894)* status: 0 of 25 files reviewed, 1 unresolved discussion (waiting on @dimaopen, @jlaz, and @nikolayilyin)

___
*[`src/main/python/freight/freight_events_processing.py` line 253 at r1](https://reviewable.io/reviews/LBNL-UCB-STI/beam/3894#gh-1804157520:-O9RtS0xBYF1zHcsFZnQ:bb6un1j) ([raw file](https://github.com/LBNL-UCB-STI/beam/blob/d4374fdb5f8ff4d03704354cad9caab13bb5c7c1/src/main/python/freight/freight_events_processing.py#L253)):*
<details><summary><i>Previously, dimaopen (Dmitrii Openkov) wrote…</i></summary><blockquote>

Older version of python (i.e. 3.10) don't support nesting same quoted strings. I suggest replace nested double-quoted strings to single quoted ones. It helps avoiding issues when running this script in other environments. 
</blockquote></details>

Done

File: src/main/python/freight/freight_events_processing.py (hunk 1 above)
Comment (haitamlaarabi) [reply src/main/python/freight/freight_events_processing.py]: Done

**Comment by haitamlaarabi** [comment]
Test!

**Comment by haitamlaarabi** [comment]
Test now!

**Comment by beambot2** [comment]
test!

**Comment by beambot2** [comment]
test now!

**Comment by beambot2** [comment]
test!

**Comment by wrashid** [comment]
test!

**Comment by wrashid** [comment]
test!

**Pull request merged**